### PR TITLE
docs(recipe): remove TableCard recipe

### PR DESCRIPTION
**Pull request by booc0mtaco**
### Summary:

- this recipe, as-is, is tightly coupled to one particular project and can't be repurposed
- move it to the one page in storybook where it is used for now

### Test Plan:

- n/a (documentation change)

### CODE REVIEW

**Comment by booc0mtaco** [comment]
@jinlee93 one more to finish this milestone out

File: .storybook/pages/CoursePlannerEdit/CoursePlannerEdit.module.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import '../../../src/design-tokens/mixins.css';
 /*-----------------------------------------*\
     # SPACING COURSE PLANNER EDIT CLASSES
```
Comment (jinlee93) [inline .storybook/pages/CoursePlannerEdit/CoursePlannerEdit.module.css:3]: nit: LOL why is it `SPACING`

**Review by jinlee93 — APPROVED** [review]
🏀 🗑️  thanks for all the clean up!

File: .storybook/pages/CoursePlannerEdit/CoursePlannerEdit.module.css (hunk 1 above)
Comment (booc0mtaco) [reply .storybook/pages/CoursePlannerEdit/CoursePlannerEdit.module.css:3]: lol wut :P I'll patch that

**Pull request merged**